.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,68 @@
+---
+name: 🐛 Bug Report
+about: Report erros and unexpected behavior
+title: 'My descriptive bug title'
+labels: ''
+assignees: ''
+
+---
+
+<!--
+
+Thank you for using Pester and taking the time to report this issue!
+
+Please be aware that Pester version 3.4.0 - which is shipped with Windows 10 and Windows Server 2016 - is not supported anymore.
+
+- Please update Pester and retest your code before submitting a bug report. See [Installation and update guide](https://pester.dev/docs/introduction/installation).
+- Search for existing issues.
+- Pester 5 introduced breaking changes and some features were removed or are not yet migrated. See [Breaking changes](https://github.com/pester/Pester#breaking-changes)
+
+-->
+
+## General summary of the issue
+
+
+## Describe your environment
```

Comment:
Link to the same script as above. Probably just the invoke-webrequest and link to gist is enough, so we don't have to manually update it in multiple places.